craft_parts/packages/apt_cache.py

```diff
@@ -246,6 +246,7 @@ def fetch_archives(self, download_path: Path) -> List[Tuple[str, str, Path]]:
                 continue
 
             try:
+                logger.info("Downloading package: %s", package)
```

Comment:
It may be also achieved by increasing verbosity in here:

https://github.com/dariuszd21/craft-parts/blob/0ee0002926bfcdd7928075026a4dd3d1defabab2/craft_parts/packages/apt_cache.py#L74-L78